skorch/tests/test_net.py

```diff
@@ -1465,22 +1464,15 @@ def __init__(self, *args, foo, **kwargs):
             dataset=MyDataset,
             dataset__foo=123,
             max_epochs=1,
-            callbacks__train_loss__use_caching=use_caching,
-            callbacks__valid_loss__use_caching=use_caching,
-            callbacks__valid_acc__use_caching=use_caching,
         )
         net.fit(*data)
+        assert side_effect == [123]
 
-        if not use_caching:
-            # train/valid split, scoring predict
-            assert side_effect == [123, 123]
-        else:
-            # train/valid split
-            assert side_effect == [123]
-
+    @pytest.mark.skip
     @pytest.mark.xfail
     def test_net_initialized_with_initalized_dataset(
             self, net_cls, module_cls, data, dataset_cls):
+        # TODO: What to do of this test now??
```

Comment:
Do we need an issue to track this?

Reply:
tbh I don't remember why I put this here but the test xfails with the expected error, so I removed the comment and skip